style.css

```diff
@@ -4,26 +4,83 @@ body {
 
 .workspace {
     position: relative;
-    width: 80%;
-    height: 80%;
+    width: 85%;
+    height: 100%;
     margin: auto;
     background-color: white;
     z-index: 1;
 }
 .color-palet {
     position: relative;
-    width: 80px;
-    left: 0%;
-    margin: 1px;
+    text-align: center;
+    width: 300px;
+    top: 100px;

```

Comment:
The position of an element is usually determined by the spacing between it an other elements, so in general you should need to use properties like `top` and `left` and instead use `margin` for spacing between the element and outside elements and `padding` for the spacing between the element and its children.